### PR TITLE
add publish pipeline on tag

### PR DESCRIPTION
**Pull request by adrian-goe**

### CODE REVIEW

File: .github/workflows/publish.yml [hunk 1]
```diff
@@ -1,13 +1,21 @@
...
     steps:
       - uses: actions/checkout@v2
-      # TODO
-#      - - name: NPM Publish
-#          uses: JS-DevTools/npm-publish@v1
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '14.x'
+          registry-url: 'https://npm.pkg.github.com'
+      - run: npm install
```
Comment (geisterfurz007) [inline .github/workflows/publish.yml:18]:
```suggestion
      - run: npm ci
```

**Review by geisterfurz007 — CHANGES_REQUESTED** [review]

File: .github/workflows/publish.yml (hunk 1 above)
Comment (adrian-goe) [reply .github/workflows/publish.yml:18]: If `yarn.lock` work there I can do this, otherwise we have to add the package-lock.json

Comment (sklirg) [reply .github/workflows/publish.yml:18]: This should then be `yarn` and not `npm install`, though?

Comment (geisterfurz007) [reply .github/workflows/publish.yml:18]: Oh shit, yes absolutely! `yarn install --frozen-lockfile` should be the equivalent, then, I believe.

**Review by sklirg — APPROVED** [review]

**Comment by adrian-goe** [comment]
Closes https://github.com/Yes-Theory-Fam/database/issues/2

**Review by geisterfurz007 — APPROVED** [review]

**Pull request merged**